fastpass/src/main/scala/scala/meta/internal/fastpass/bazelbuild/Bazel.scala

```diff
@@ -449,4 +464,40 @@ class Bazel(bazelPath: Path, cwd: Path) {
       .exitValue()
   }
 
+  def runWithErrorCatching(
+      cmd: List[String],
+      out: OutputStream,
+      err: OutputStream = FileUtils.NullOutputStream,
+      exc: String
+  ): Int = {
+    val outByteArrayOutputStream = new ByteArrayOutputStream
+    val errByteArrayOutputStream = new ByteArrayOutputStream
+    val outPrintStream = new PrintStream(outByteArrayOutputStream)
+    val errPrintStream = new PrintStream(errByteArrayOutputStream)
+    val io =
+      new ProcessIO(
+        _ => (),
+        FileUtils.copy(_, outPrintStream),
+        FileUtils.copy(_, errPrintStream)
+      )
+
+    val errWriter = new PrintWriter(err)
+    errWriter.println(s"# Running command in ${cwd}:")
+    errWriter.println(s"$$ ${cmd.mkString(" ")}")
+    val exitCode = Process(cmd, cwd.toFile)
+      .run(io)
+      .exitValue()
+
+    out.write(outByteArrayOutputStream.toByteArray)
+    err.write(errByteArrayOutputStream.toByteArray)
+
+    if (
+      outByteArrayOutputStream.toString().contains(exc)
+      || errByteArrayOutputStream.toString().contains(exc)
```

Comment:
This is because I am not sure if running process will always redirect exception on `err` or is it possible to redirect `FATAL: bazel crashed due to an internal error.` also on `out` (while I am debugging it is always `err` but I thought it would be better to ask about it)

Reply:
I think it should be okay to assume that we only need to look at stderr, because these functions are meant to be calling Bazel, which does a good job of writing to the right stream